app/src/main/java/com/strv/movies/ui/navigation/MoviesNavGraph.kt

```diff
@@ -0,0 +1,46 @@
+package com.strv.movies.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.strv.movies.data.OfflineMoviesProvider
+import com.strv.movies.ui.moviedetail.MovieDetail
+import com.strv.movies.ui.movieslist.MoviesList
+
+@Composable
+fun MoviesNavGraph(
+    navController: NavHostController = rememberNavController()
+) {
+    NavHost(
+        navController = navController,
+        startDestination = MoviesDestinations.MOVIES_LIST_ROUTE
+    ) {
+        composable(MoviesDestinations.MOVIES_LIST_ROUTE) {
+            MoviesList(
+                movies = OfflineMoviesProvider.getMovies(),
+                onMovieClick = { movieId ->
+                    navController.navigate("${MoviesDestinations.MOVIE_DETAIL_ROUTE}/$movieId")
+                }
+            )
+        }
+
+        composable(
+            route = "${MoviesDestinations.MOVIE_DETAIL_ROUTE}/{${MoviesNavArguments.MOVIE_ID_KEY}}",
+            arguments = listOf(
+                navArgument(MoviesNavArguments.MOVIE_ID_KEY) {
+                    type = NavType.IntType
+                }
+            )
+        ) { backStackEntry ->
+            val movieId = backStackEntry.arguments?.getInt(MoviesNavArguments.MOVIE_ID_KEY)
+
+            movieId?.let {
+                MovieDetail(movie = OfflineMoviesProvider.getMovieDetail(movieId))
+            }
+        }
+    }
+}
```

Comment:
Looking good generally. Some part of this could be copy-paste snippet or something. I would add some comments in code so it is more comprehendable. Comments could be also added elsewhere.